src/table/__tests__/table-feature-metrics.test.tsx

```diff
@@ -0,0 +1,50 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import * as React from 'react';
+import { render } from '@testing-library/react';
+
+import * as baseComponentHooks from '../../../lib/components/internal/hooks/use-base-component';
+import Table from '../../../lib/components/table';
+
+const useBaseComponentSpy = jest.spyOn(baseComponentHooks, 'default');
+
+jest.mock('../../../lib/components/internal/hooks/use-mobile', () => ({
+  useMobile: jest.fn(),
+}));
```

Comment:
Why is this mock needed?

Evergreen rule – less mocks, better tests

See this PR for an example why mocks are bad: https://github.com/cloudscape-design/components/pull/3277

Reply:
That was a copy-paste artifact from the table tests file, it is not needed for the metrics tests - removed.